scene/3d/node_3d.h

```diff
@@ -133,7 +133,7 @@ class Node3D : public Node {
 		bool transform_gizmo_visible = true;
 #endif
 
-	} data;
+	} data; // This avoids namespace pollution in large classes, do not remove.
```

Comment:
I'd put it before the struct. Long inline comments are often awkward.
I'd phrase it:

> This Data struct is to avoid namespace pollution in derived classes.

Reply:
Sounds good to me. Changed to that.

Reply:
> This Data struct is to avoid namespace pollution in derived classes.

`data` is `private` so it has nothing to do with derived classes, it avoids polutting the class own namespace (the same for each commented `data`).

Reply:
Welp, missed this comment before merging. Can be tweaked further in a future PR... but ideally waiting for one that actually *needs* to modify one of these core headers, the CI is crying when it sees cosmetic changes in core headers.